src/settings/appsettings.cpp

```diff
@@ -986,6 +986,33 @@ bool AppSettings::defaultShowMagnifier()
     return false;
 }
 
+QMap<QString, QVariant> AppSettings::tesseractParameters() const
+{
+    QMap<QString, QVariant> parameters;
+    m_settings->beginGroup("Tesseract");
+    QStringList keys = m_settings->childKeys();
+    for (const auto &key : keys) {
+        parameters.insert(key, m_settings->value(key));
+    }
+    m_settings->endGroup();
+    return parameters;
+}
+
+void AppSettings::setTesseractParameters(const QMap<QString, QVariant> &parameters)
+{
+    m_settings->beginGroup("Tesseract");
+    m_settings->remove("");
+    for (auto it = parameters.cbegin(); it != parameters.cend(); ++it) {
+        m_settings->setValue(it.key(), it.value());
+    }
+    m_settings->endGroup();
+}
+
+QMap<QString, QVariant> AppSettings::defaultTesseractParameters()
+{
+    return QMap<QString, QVariant>();
```

Comment:
I've used `{}` in this file to return the default. So, just for consistency.

```suggestion
    return {};
```